lib/ProductOpener/Images.pm

```diff
@@ -311,6 +311,25 @@ HTML
 
 
 	$scripts .= <<JS
+<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/jsbarcode@3.11.0/dist/JsBarcode.all.min.js"></script>
+<script type="text/javascript">
+// This code will execute only when javascript is enabled.
+
+// Get the barcode `span` element from DOM.
+const barcode = document.getElementById('barcode');
+
+// Get the barcode_paragraph `p` element from DOM.
+const barcodeParagraph = document.getElementById('barcode_paragraph');
```

Comment:
The `barcode_paragraph` element does not exist, if the barcode doesn't start with a `0`, so that should likely be checked before doing all of this. Maybe even the `JsBarcode.all.min.js` script only needs to be included, if we know the barcode will be replaced by an SVG.

Reply:
Also, it may need to be limited to some page types, because right now, the barcode is also replaced if I still have to log in: ![image](https://user-images.githubusercontent.com/87124/93685521-43fc0780-fab0-11ea-8348-c7efd26b6d8b.png)

Reply:
Can you mention the pages, on which the `barcode` is needed

Reply:
@stephanegigandet @teolemon Where did you initially want the barcode to be displayed?

Reply:
No one is replying 🙁

Reply:
Hi @ArishSultan, sorry for my late reply. We should show the barcode only the product info page: https://world.openfoodfacts.org/product/5022019160310/organic-ginger-beer-belvoir

Ideally it would appear when we click on a barcode button.

Reply:
Ok, i am on it